### DISCUSSION TITLE
Locode

### DISCUSSION THREAD

**Discussion by dhaanpaa-lab0** [ServiceStack/Discuss#20] [General]
First of all, what i have seen so far is awesome. How do I add authentication to my Locode.dev app? Thank you so much.

**Comment by mythz** [comment]
Locode is just a UI around ServiceStack APIs, so you just configure your App with ServiceStack Auth as normal:
https://docs.servicestack.net/authentication-and-authorization

E.g. this is the Auth configuration of the https://talent.locode.dev Demo:
https://github.com/NetCoreApps/TalentBlazor/blob/main/TalentBlazor/Configure.Auth.cs
